src/schema/rules/sidecars/mrs.yaml

```
@@ -0,0 +1,98 @@
+#
+# Groups of related metadata fields
+#
+# Assumptions: never need disjunction of selectors
+# Assumptions: top-to-bottom overrides is sufficient logic
+
+---
+# MRS Common metadata fields
+MRSScannerHardware:
+  selectors:
+    - modality == "mrs"
+  fields:
+    InstitutionName: recommended
+    InstitutionAddress: recommended
+    InstitutionalDepartmentName: recommended
+    Manufacturer: recommended
+    ManufacturersModelName: recommended
+    DeviceSerialNumber: recommended
+    StationName: recommended
+    SoftwareVersions: recommended
+    MagneticFieldStrength: recommended
+    ReceiveCoilName: recommended
+    ReceiveCoilActiveElements: recommended
+    NumberReceiveCoilActiveElements: optional
+    NumberTransmitCoilActiveElements: optional
+
+MRSSample:
+  selectors:
+    - modality == "mrs"
+  fields:
+    BodyPart:
+      level: optional
+      description_addendum: Corresponds to DICOM Tag 0018, 0015 `Body Part Examined`.
+    BodyPartDetails: optional
+    BodyPartDetailsOntology: optional
+
+MRSSequenceSpecifics:
+  selectors:
+    - modality == "mrs"
+  fields:
+    PulseSequenceType:
+      level: recommended
+      description_addendum: |
+        For example, `"sLASER"`, `"MEGA-PRESS"`, `"EPSI"`, `"Metabolite-cycled MRSI"`.
+    ScanningSequence__mrs: recommended
+    SequenceName: recommended
+    PulseSequenceDetails: recommended
+    MRAcquisitionType: recommended for MRSI
```

I think this would be:

```suggestion
    MRAcquisitionType: optional, but recommended for MRSI
```

Then we should add an additional table:

```
MRSIRecommendedFields:
  selectors:
    - modality == "mrs"
    - suffix == "mrsi"
  fields:
    MRAcquisitionType: recommended
```

We can add the other ones from MRSRecommendedFields, as well.